recipe/meta.yaml

```diff
@@ -30,10 +30,10 @@ requirements:
     - wheel
     - setuptools
   run:
-    - python >=3.6
+    - python
     - botocore {{ botocore_bounds }}
     - jmespath >=0.7.1,<2.0.0
-    - s3transfer >=0.5.0,<0.6.0
+    - s3transfer >=0.6.0,<0.7.0
 
 test:
   imports:
```

Comment:
doc_url should be: https://boto3.amazonaws.com/v1/documentation/api/latest/index.html